chart/templates/service.yaml

```diff
@@ -13,5 +13,9 @@ spec:
     targetPort: 80
     protocol: TCP
     name: http
+  - port: 443
+    targetPort: 444
+    protocol: TCP
+    name: https-internal
```

Comment:
This is different from other PR, was there an additional PR that added this stuff into master earlier ?

Reply:
@cbron Yeah, there are two commits made